apps/examples/testcase/le_tc/network/Kconfig

```diff
@@ -36,6 +36,9 @@ config TC_NET_ALL
 	select TC_NET_INET
 	select TC_NET_ETHER
 	select TC_NET_NETDB
+	select TC_NETDEV_CARRIER
+	select TC_NET_DECISEC_TIME
+        select TC_NET_CORE
```

Comment:
Please use a same method for indentation. Tab is used at above, but space is here.